src/Argu/Attributes.fs

```diff
@@ -128,6 +129,30 @@ type EqualsAssignmentAttribute () =
 [<AttributeUsage(AttributeTargets.Property, AllowMultiple = false)>]
 type ColonAssignmentAttribute () = 
     inherit CustomAssignmentAttribute(":")
+    
+/// Use an custom separator for parameter assignment.
+/// e.g. '--param<separator>arg' or '--param key<separator>value'.
+/// Parameters can also be assigned using space as separator e.g. '--param arg'
+/// Requires that the argument should have parameters of arity 1 or 2 only.
+/// Can be used to specify any assignment separator.
+[<AttributeUsage(AttributeTargets.Property, AllowMultiple = false)>]
+type EitherSpaceOrCustomAssignmentAttribute (separator : string) = 
```

Comment:
I find the attribute name to be too long. How about `CustomAssignmentOrSpaced`, since it's ultimately determined by whitespace between argv and not a literal space character that Argu handles.